src/client/array/dc_array.c

```diff
@@ -189,7 +189,10 @@ create_handle_cb(tse_task_t *task, void *data)
 	int			rc = task->dt_result;
 
 	if (rc != 0) {
-		D_ERROR("Failed to create array obj "DF_RC"\n", DP_RC(rc));
+		if (rc == -DER_EXIST)
```

Comment:
can use D_CDEBUG(), just FYI.

Reply:
ah yes forgot about that.  updated.
please review again